doc/advanced_usage.md

```diff
@@ -92,4 +92,7 @@ the status would be 400.
 
 ## Spectroscopic lines
 
-The line list implemented on SkyPortal was derived using the [NIST atomic database](https://www.nist.gov/pml/atomic-spectra-database), employing the methods of [Gal-Yam  2019](https://ui.adsabs.harvard.edu/abs/2019ApJ...882..102G/abstract).
+The selection of element and galaxy lines on SkyPortal were derived from the NIST atomic database (https://www.nist.gov/pml/atomic-spectra-database) , using the methods of Gal-Yam (2019) (https://ui.adsabs.harvard.edu/abs/2019ApJ...882..102G/abstract). 
+Telluric and Skylines were migrated from the ZTF Marshal. 
```

Comment:
Use < >  brackets for the URLs to make them work as links. 
<https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>

Reply:
Also I think you should try leaving an empty line at the end of the file to pass some of the "linting" tests.

Reply:
Right, forgot about this. Fixing it.